experimenter/experimenter/test.txt

```diff
@@ -0,0 +1 @@
+test
```

Comment:
do we want this file?

Reply:
That's just to force it to run the entire suite which is gated on a path check I'll remove it before merging